src/components/src/AppHeader/appheader.main.less

```diff
@@ -41,6 +41,15 @@
     fill: @mainTextColor;
   }
 
+  .impersonation-notification {
+    height: 100%;
+    background-color: #8b1e24;
```

Comment:
Now that we're re-visiting this, where does this color code come from? It's not currently in line with our color palette. 

Will merge this for now since it's needed.

Reply:
![shopping-impersonation-notification](https://user-images.githubusercontent.com/25517396/75173282-7c3a9380-56e3-11ea-914c-d1ddbc209589.PNG)

Here is a picture of the banner.  I am not sure where the color originally came from, but we are totally open to changing it.  At the very least we should probably move it to common.less

Reply:
When the idea of having a shopper impersonation banner was introduced, the criteria was to have a red banner so it was very obvious this was shopper impersonation. I picked this shade of red myself, but open to changing the colour. I agree this should be moved to `common.less`